### PR TITLE
MemberListCard UI Fix

### PR DESCRIPTION
**Pull request by GLundh**
When a description is provided and the user hovers over the email link, the description jumps a bit in the UI.

Signed-off-by: Gustaf Lundh <gustaf.lundh@axis.com>

https://user-images.githubusercontent.com/300900/217842034-d8fb4f94-2be3-4403-b8a1-fc5c2aed39fe.mp4

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: package.json
Comment (awanlin) [inline package.json:76]: Pretty sure this isn't needed for your change, if you undo this then you'd avoid changes to the `yarn.lock` as well

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this fix @GLundh! Just left a small comment, your actual fix LGTM 🚀

File: package.json
Comment (awanlin) [reply package.json]: No worries!

**Review by awanlin — APPROVED** [review]

**Comment by GLundh** [comment]
Is there anything I need to to do after @awanlin 's approval?

**Comment by awanlin** [comment]
@GLundh the maintainers have been pretty busy getting some big things in place. They seem a bit more free now so let's give them a ping.

@freben any chance you could give this a quick review and approval, please? I've tested this out myself and it works nicely.

**Comment by freben** [comment]
Another reason is also that we just changed our workflows significantly, switching from assignment to instead using the reviewers feature of github itself - and I'm ashamed to say that me looking in my old regular places instead of the new one made things go below my radar 😨 I'll get on it!

**Review by freben — APPROVED** [review]
Thanks!

**Pull request merged**
